themes/survey/fruity_twentythree/views/layout_survey_list.twig

```diff
@@ -176,20 +176,20 @@ NOTE: Feel free to ask for functions, variables, etc, that you feel are missing
                                 </div>
                             </div>
                         </div>
-                    </div>
+                    </main>
 
-                    <div id="{{ aSurveyInfo.id.surveylistfooter }}"
+                    <footer role="contentinfo" id="{{ aSurveyInfo.id.surveylistfooter }}"
                          class="{{ aSurveyInfo.class.surveylistfooter }}" {{ aSurveyInfo.attr.surveylistfooter }}>
                         <div
                             class="{{ aSurveyInfo.class.surveylistfootercont }}" {{ aSurveyInfo.attr.surveylistfootercont }}>
                             <div {{ aSurveyInfo.attr.surveylistfootercontp }} class="row">
                                 <div class="col-6 col-md-12">
                                     <a {{ aSurveyInfo.attr.surveylistfootercontpaa }}>
-                                        {{ image("./files/poweredby.png", "LimeSurvey Survey Software", {class: 'img-fluid'}) }}
+                                        {{ image("./files/poweredby.png", "proudly powered by LimeSurvey", {class: 'img-fluid'}) }}
```

Comment:
gT ?

Reply:
Yes, this needs to be put into translation.

Reply:
Done